path_role.go

```diff
@@ -289,9 +289,10 @@ func (b *kubeAuthBackend) pathRoleCreateUpdate(ctx context.Context, req *logical
 		return logical.ErrorResponse("token ttl should not be greater than token max ttl"), nil
 	}
 
-	var resp *logical.Response
+	resp := &logical.Response{}
```

Comment:
Is this going to change the return code of the create happy path now that resp is always non-nil?